From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Raine Makelainen <raine.makelainen@jolla.com>
Date: Thu, 14 Jan 2021 12:27:09 +0200
Subject: [PATCH] [sailfishos][gecko] Ignore safemode in gfxPlatform. Fixes
 JB#52738

This is needed in order make/keep rendering pipeline working on Sailfish OS
when safemode is enabled.

See EmbedLiteXulAppInfo::GetInSafeMode and related change.

Signed-off-by: Raine Makelainen <raine.makelainen@jolla.com>
---
 gfx/thebes/gfxPlatform.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gfx/thebes/gfxPlatform.cpp b/gfx/thebes/gfxPlatform.cpp
index c2fd5b3649c6..8de39356df84 100644
--- a/gfx/thebes/gfxPlatform.cpp
+++ b/gfx/thebes/gfxPlatform.cpp
@@ -2051,7 +2051,7 @@ BackendType gfxPlatform::GetBackendPref(const char* aBackendPrefName,
 }
 
 bool gfxPlatform::InSafeMode() {
-  static bool sSafeModeInitialized = false;
+  static bool sSafeModeInitialized = true;
   static bool sInSafeMode = false;
 
   if (!sSafeModeInitialized) {
-- 
2.31.1

